src/main/java/nextstep/subway/station/dto/StationResponse.java

```diff
@@ -1,42 +1,53 @@
 package nextstep.subway.station.dto;
 
-import nextstep.subway.station.domain.Station;
-
 import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.station.domain.Station;
 
 public class StationResponse {
-    private Long id;
-    private String name;
-    private LocalDateTime createdDate;
-    private LocalDateTime modifiedDate;
-
-    public static StationResponse of(Station station) {
-        return new StationResponse(station.getId(), station.getName(), station.getCreatedDate(), station.getModifiedDate());
-    }
-
-    public StationResponse() {
-    }
-
-    public StationResponse(Long id, String name, LocalDateTime createdDate, LocalDateTime modifiedDate) {
-        this.id = id;
-        this.name = name;
-        this.createdDate = createdDate;
-        this.modifiedDate = modifiedDate;
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public LocalDateTime getCreatedDate() {
-        return createdDate;
-    }
-
-    public LocalDateTime getModifiedDate() {
-        return modifiedDate;
-    }
+	private Long id;
+	private String name;
+	private LocalDateTime createdDate;
+	private LocalDateTime modifiedDate;
+
+	public StationResponse() {
+	}
+
+	public StationResponse(Long id, String name, LocalDateTime createdDate, LocalDateTime modifiedDate) {
+		this.id = id;
+		this.name = name;
+		this.createdDate = createdDate;
+		this.modifiedDate = modifiedDate;
+	}
+
+	public static StationResponse of(Station station) {
+		return new StationResponse(station.getId(), station.getName(), station.getCreatedDate(),
+			station.getModifiedDate());
+	}
+
+	public static List<StationResponse> ofList(List<Station> stations) {
+		return stations.stream()
+			.map(station -> StationResponse.of(station))
+			.collect(Collectors.toList());
+	}
```

Comment:
캡슐화 잘해주셨네요 👍 
`메소드 레퍼런스`를 이용해볼 수 있겠네요 :)

```suggestion
	public static List<StationResponse> ofList(List<Station> stations) {
		return stations.stream()
			.map(StationResponse::of)
			.collect(Collectors.toList());
	}
```